src/client/app/components/UIOptionsComponent.tsx

```diff
@@ -46,10 +46,13 @@ export default class UIOptionsComponent extends React.Component<UIOptionsProps,
 		this.formatSliderTip = this.formatSliderTip.bind(this);
 		this.handleBarButton = this.handleBarButton.bind(this);
 		this.handleCompareButton = this.handleCompareButton.bind(this);
+		this.handleSortingButton = this.handleSortingButton.bind(this);
 		this.toggleSlider = this.toggleSlider.bind(this);
+		this.toggleDropdown = this.toggleDropdown.bind(this);
 		this.state = {
 			barDurationDays: this.props.barDuration.asDays(),
-			showSlider: false
+			showSlider: false,
+			dropdownOpen: false
```

Comment:
Again, just nitpicking on variable names. I would call this `compareSortingDropdown`.